week2/homework/exercises/1-blog-API/server.js

```diff
@@ -1,10 +1,68 @@
-const express = require('express')
+'use strict';
+const express = require("express");
 const app = express();
- 
 
 // YOUR CODE GOES IN HERE
-app.get('/', function (req, res) {
-  res.send('Hello World')
-})
- 
+// READ    - GET     -/blogs
+// CREATE  - POST    -/blogs
+// UPDATE  - PUT     -/blogs/:title
+// DELETE  - DELETE  -/blogs/:title
+
+app.use(express.json()); //for parsing json in request body
+const fs = require("fs");
+
+// creatBook
+app.post("/blogs", (req, res) => {
+  // How to get the title and content from the request??
+  const title = req.body.title;
+  const content = req.body.content;
+  fs.writeFileSync(title, content);
+  res.end("ok");
+});
+
+//updateBook
+app.put("/blogs/:title", (req, res) => {
+
+  //to check if a file exists.
+  // fs.existsSync(req.body.title)
+  // console.log(req.body.title);
+
+  //getting data
+  const title = req.body.title;
+  const content = req.body.content;
+
+  // getting title from the url parameters
+  const searchedTitle =req.params.title
+
+  if (title === searchedTitle) {
+    fs.writeFileSync(title, content);
+    res.end(`Ok ${title} is updated.`);
+  } else {
+    res.end("This post does not exist!");
+    return;
+  }
+});
+
+//deleteBook
+app.delete("/blogs/:title", (req, res) => {
+  const URLTitle = req.params.title;
```

Comment:
```suggestion
  const urlTitle = req.params.title;
```
`camelCase` is the standard for variables in JS

Reply:
I changed it with urlTitle